.github/workflows/docker.yml

```diff
@@ -71,3 +71,12 @@ jobs:
             ${{ env.REGISTRY }}/${{ secrets.DOCKERHUB_USERNAME }}/${{ env.IMAGE_NAME }}:${{ github.sha }}
           cache-from: type=gha
           cache-to: type=gha,mode=max
+
+      - name: Update Docker Hub description + short desc
```

Comment:
The step uses repository secrets but is not guarded. To ensure it only runs when publishing to Docker Hub (e.g., on push to main) and to avoid secret access attempts on PRs from forks, add an explicit if condition at the step level (for example: if: github.event_name == 'push' && github.ref == 'refs/heads/main').